mediathread/main/views.py

```diff
@@ -891,6 +891,7 @@ def create_course(self, form, affil):
 
         # Add the current user as an instructor.
         c.faculty_group.user_set.add(self.request.user)
+        c.add_detail('instructor', self.request.user.profile.fullname)
```

Comment:
We are displaying instructor names with `User.get_full_name` on the CourseListView: https://github.com/ccnmtl/mediathread/blob/master/mediathread/templates/courseaffils/course_row.html#L23  Should I use that instead of `get_public_name`?